content-keys-test-vectors.md

```diff
@@ -0,0 +1,117 @@
+# Content Keys Test Vectors
+
+This document contains test vectors for content keys to help implementations conform to the specification. 
+
+## Content Keys Encodings
```

Comment:
Can we have somewhere a split of sections between the state content keys and the history content keys? That would make it more readable/searchable/linkable.

Reply:
And once that is there, links could be added to the test-vectors in the state and history specs